tests/Unit/Evolution/EventsAndDenseTriggers/DenseTriggers/Test_Times.cpp

```diff
@@ -69,8 +69,12 @@ void check_one_direction(const std::vector<double>& trigger_times,
   const void* component = nullptr;
 
   CHECK(trigger->is_ready(box, cache, array_index, component));
+  CHECK_FALSE(trigger->previous_trigger_time().has_value());
   const auto is_triggered = trigger->is_triggered(box);
   CHECK(is_triggered.is_triggered == expected_is_triggered);
+  if (expected_is_triggered) {
+    CHECK(trigger->previous_trigger_time().value() == current_time);
+  }
```

Comment:
Check that it didn't get set in the other case.  If this code is obsoleted by changes to `previous_trigger_time()` in the other commit you can ignore this.